src/common/components/menuDropdown/__tests__/MenuDropdown.test.tsx

```diff
@@ -0,0 +1,165 @@
+import { IconPen } from 'hds-react';
+import React from 'react';
+
+import {
+  arrowDownKeyPressHelper,
+  arrowUpKeyPressHelper,
+  configure,
+  enterKeyPressHelper,
+  escKeyPressHelper,
+  render,
+  screen,
+  tabKeyPressHelper,
+  userEvent,
+  waitFor,
+} from '../../../../utils/testUtils';
+import MenuDropdown, { MenuDropdownProps } from '../MenuDropdown';
+import { MenuItemOptionProps } from '../types';
+
+configure({ defaultHidden: true });
+
+const renderMenuDropdown = (props: MenuDropdownProps) => {
+  render(<MenuDropdown {...props} />);
+
+  const getItemAtIndex = (index: number) =>
+    screen.getByRole('button', { name: items[index].label });
+
+  return {
+    getItemAtIndex,
+    getElement,
+  };
+};
+
+const getElement = (key: 'toggleButton') => {
+  switch (key) {
```

Comment:
With just one value to match the `switch` statement seems unnecessary for the current implementation (although it looks like it has been used in a similar way elsewhere too).

Reply:
I have started to refactor switch statements with less that 3 cases. I'll change them when they are related to the current PR. Missed this one but it's changed now.